apps/molgenis-components/src/components/filters/FilterInput.vue

```diff
@@ -16,15 +16,36 @@
 
 <script>
 import StringFilter from "./StringFilter.vue";
+import IntegerFilter from "./IntegerFilter.vue";
+import DecimalFilter from "./DecimalFilter.vue";
 
 export default {
   name: "FilterInput",
-  components: { StringFilter },
+  components: { StringFilter, IntegerFilter },
   props: {
     id: {
       type: String,
       required: true,
     },
+    columnType: {
+      type: String,
+      required: true,
+      validator(value) {
+        // The value must match one of these strings
```

Comment:
the code is quite clear, so this comment feels unneeded to me

Reply:
yes will remove it , left over from code before adding the validator